website/docs/components/processors/nats_kv.md

```diff
@@ -225,6 +226,26 @@ revision: "42"
 revision: ${! @nats_kv_revision }
 ```
 
+### `keys_filter`
```

Comment:
I'm going back and forth with the idea of introducing this vs reusing the `key` field. My hangup is that at the end of the day they are serving the same purpose for any `watch` related options, so `History`, `Keys` and `Watch` in the input. I wonder if the `key` field should be used for all of it. WDYT

Reply:
Yeah, I'm leaning towards that as well. My initial thought was that it might be confusing for users if they set a wildcard for operations which don't support them, but they should get an error in that case anyway. I'll just document that the `history` and `keys` operations support wildcards. Should've done this from the start.

Reply:
Yeah. Looking at the code I'm leaning more toward that! Nice call